src/fec_rs_fnt.h

```diff
@@ -83,18 +82,25 @@ class RsFnt : public FecCode<T> {
         n = this->gf->get_code_len_high_compo(n_parities + n_data);
 
         // compute root of order n-1 such as r^(n-1) mod q == 1
-        r = this->gf->get_nth_root(n);
+        this->r = this->gf->get_nth_root(n);
 
         // std::cerr << "n=" << n << "\n";
         // std::cerr << "r=" << r << "\n";
 
         int m = arith::get_smallest_power_of_2<int>(n_data);
         this->fft = new fft::Radix2<T>(this->gf, n, m, pkt_size);
+
+        // vector stores r^{-i} for i = 0, ... , k
+        T inv_r = this->gf->inv(this->r);
+        this->inv_r_powers = std::unique_ptr<vec::Vector<T>>(
```

Comment:
I don't now if we want to do it, but if we bump from C++11 to C++14 (I did it for Quadiron, since C++14 is supported since several version for both GCC and Clang), you could use std::make_unique here.

Anyway, not to be done in this PR.

Reply:
yes, thanks. I noticed it for C++14 or later :)